### PR TITLE
Merge 2.3 into master

### PR DESCRIPTION
**Pull request by jjanvier**
Yeah \o/

### CODE REVIEW

File: .ci/builder.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 steps:
   - name: 'gcr.io/cloud-builders/docker'
-    args: ['pull', 'eu.gcr.io/akeneo-ci/pim-community-dev:master']
+    args: ['pull', 'eu.gcr.io/akeneo-ci/pim-community-dev:2.3']
   - name: 'gcr.io/cloud-builders/docker'
-    args: ['build', '--cache-from', 'eu.gcr.io/akeneo-ci/pim-community-dev:master', '-t', '${_IMAGE_TAG}', '.']
+    args: ['build', '--cache-from', 'eu.gcr.io/akeneo-ci/pim-community-dev:2.3', '-t', '${_IMAGE_TAG}', '.']
```
Comment (damien-carcel) [inline .ci/builder.yaml]: No, you need to keep master here. It is the master branch.

Comment (jjanvier) [reply .ci/builder.yaml]: thanks!

File: .ci/builder.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 steps:
   - name: 'gcr.io/cloud-builders/docker'
-    args: ['pull', 'eu.gcr.io/akeneo-ci/pim-community-dev:master']
+    args: ['pull', 'eu.gcr.io/akeneo-ci/pim-community-dev:2.3']
```
Comment (juliensnz) [inline .ci/builder.yaml]: are we sure about that?

File: .ci/builder.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ steps:
   - name: 'gcr.io/cloud-builders/docker'
     args: ['pull', 'eu.gcr.io/akeneo-ci/pim-community-dev:master']
   - name: 'gcr.io/cloud-builders/docker'
-    args: ['build', '--cache-from', 'eu.gcr.io/akeneo-ci/pim-community-dev:master', '-t', '${_IMAGE_TAG}', '.']
+    args: ['build', '--cache-from', 'eu.gcr.io/akeneo-ci/pim-community-dev:2.3', '-t', '${_IMAGE_TAG}', '.']
```
Comment (juliensnz) [inline .ci/builder.yaml]: are we sure about that?

Comment (jjanvier) [reply .ci/builder.yaml]: thanks, fixed

File: src/Pim/Bundle/CatalogBundle/Version.php [hunk 4]
```diff
@@ -12,10 +12,10 @@
 class Version
 {
     /** @staticvar string */
-    const VERSION = '2.3.0-BETA1';
+    const VERSION = '2.3.0';
```
Comment (damien-carcel) [inline src/Pim/Bundle/CatalogBundle/Version.php]: To change for 3.0 ?

File: src/Pim/Bundle/CatalogBundle/Version.php [hunk 5]
```diff
@@ -12,10 +12,10 @@
...
 
     /** @staticvar string */
-    const VERSION_CODENAME = '';
+    const VERSION_CODENAME = 'What’s up Doc?';
```
Comment (damien-carcel) [inline src/Pim/Bundle/CatalogBundle/Version.php]: To remove?

Comment (jjanvier) [reply src/Pim/Bundle/CatalogBundle/Version.php]: don't think it's really important
but you can remove it in one of your PR if you want

**Comment by jjanvier** [comment]
- 1 phpunit failing due to a bad usage: fix in the second to last commit
- 5 behats failing due to "missing" translations: fix in the second to last commit
- last commit fixes CS on tests/Acceptance directory

Merging

**Pull request merged**